go.mod

```diff
@@ -1,19 +1,24 @@
 module github.com/metal-stack/firewall-controller
 
-go 1.18
+go 1.17
```

Comment:
why not 1.18 ?

Reply:
There was a problem with metal-networker before. Seems like it's fixed now. Updated to go1.18